outbound.js

```diff
@@ -1144,7 +1147,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
         }
 
         // TLS
-        if (smtp_properties.tls && cfg.enable_tls && !secured) {
+        if (!(self.todo.domain in tls_config.no_tls_hosts) &&
+            !(host in tls_config.no_tls_hosts) &&
+            smtp_properties.tls && cfg.enable_tls && !secured)
+        {
+            console.log(tls_config.no_tls_hosts);
```

Comment:
Please *read* the messages. It sure **looks** like @baudehlo is still working on it. You'll also notice that it's in an -alpha release version, for early testing. Who was silly enough to claim production ready?